Split the QGIS server tab in project properties

PhilipAlbrecht opened the pull request:
<!---
Updated Screenshot in:
server_manual/getting_started

Should there be more updated?
 
A list of issues is at https://github.com/qgis/QGIS-Documentation/issues.
Add "fix #issuenumber" for each issue the PR fixes. The ticket(s) will be closed automatically.
If your PR doesn't fix entirely the ticket, only add the ticket(s) reference preceded by # character.
-->
Goal: Fixes #7261

Ticket(s): #
<!---
Indicate whether the fix should be backported to previous release.
Replace the space between square brackets by a `x` to make it checked.
-->
- [ ] Backport to LTR documentation is requested

<!---
Reviewing is a process done by community members, mostly on a volunteer basis.
We try to keep the overhead as small as possible and appreciate if you help us.
Please read carefully and ensure you comply with our writing guidelines at
https://docs.qgis.org/testing/en/docs/documentation_guidelines/index.html.
Feel free to ask in a comment or the (qgis-community-team mailing list)
[https://lists.osgeo.org/mailman/listinfo/qgis-community-team] if you have troubles with any item.
--->

DelazJ commented:
@PhilipAlbrecht While addressing a new issue, you should start your branch from master and not from the previous one you used

DelazJ commented:
I think we need more here:
- [ ] there's an update needed at https://docs.qgis.org/testing/en/docs/user_manual/introduction/qgis_configuration.html#qgis-server-properties (where I would use your screenshot, maybe with the "Test configuration" menu also expanded)
- [ ] the screenshot in https://docs.qgis.org/testing/en/docs/server_manual/getting_started.html#creatingwmsfromproject which you are updating here could display the WMS tab
- [ ] the WFS tab needs renaming in the text
- [ ] (optional) Question: does it make sense to display tabs for each subsection of https://docs.qgis.org/testing/en/docs/server_manual/getting_started.html#creatingwmsfromproject? In which case the first screenshot would display the "service capabilities" + "Test configuration" menus expanded.

PhilipAlbrecht commented:
> * [ ]  there's an update needed at https://docs.qgis.org/testing/en/docs/user_manual/introduction/qgis_configuration.html#qgis-server-properties (where I would use your screenshot, maybe with the "Test configuration" menu also expanded)

- updated with expanded Test configuration

> * [ ]  the screenshot in https://docs.qgis.org/testing/en/docs/server_manual/getting_started.html#creatingwmsfromproject which you are updating here could display the WMS tab

- i did a screenshot for all tabs now

> * [ ]  the WFS tab needs renaming in the text

- which part of the text do you mean where WFS tab needs remaining. couldn't find a part where it could be mentioned in this chapter. i added a wmts theme..did mean this? 


> * [ ]  (optional) Question: does it make sense to display tabs for each subsection of https://docs.qgis.org/testing/en/docs/server_manual/getting_started.html#creatingwmsfromproject? In which case the first screenshot would display the "service capabilities" + "Test configuration" menus expanded.

- i did a screenshot for all tabs now. let me know what you think

DelazJ commented on docs/server_manual/getting_started.rst:
```diff
@@ -910,21 +918,55 @@ expressions) this will be listed inside the GetFeatureInfo output. If the
+:guilabel:`WMTS capabilities` section, QGIS Server will advertise this specific
+URL in the WMTS GetCapabilities response.
+
 WFS capabilities
 ----------------
```
```suggestion
WFS/OAPIF capabilities
----------------------
```

DelazJ commented on docs/server_manual/getting_started.rst:
```diff
@@ -910,21 +918,55 @@ expressions) this will be listed inside the GetFeatureInfo output. If the
 ----------------
 
-In the :guilabel:`WFS capabilities` area you can select the layers you
+In the :guilabel:`WFS capabilities` tab, you can select the layers you
 want to publish as WFS, and specify if they will allow update, insert and
```
```suggestion
In the :guilabel:`WFS/OAPIF capabilities` tab, you can select the layers you
want to publish as WFS or OAPIF, and specify if they will allow update, insert and
```

DelazJ commented on docs/server_manual/getting_started.rst:
```diff
@@ -910,21 +918,55 @@ expressions) this will be listed inside the GetFeatureInfo output. If the
+.. figure:: img/ows_server_wfs.png
+   :align: center
+
+   Definitions in the WFS tab
```
```suggestion
   Definitions in the WFS/OAPIF tab
```

DelazJ commented on docs/server_manual/getting_started.rst:
```diff
@@ -818,10 +821,7 @@ styles of the layers in QGIS and the project CRS, if not already defined.
-:guilabel:`Service Capabilities`.
+You have to :guilabel:`Enable Service Capabilities` first, if it is deactivated. 
 This will appear in the GetCapabilities response of the WMS, WFS or WCS.
 If you don't check |checkbox| :guilabel:`Service capabilities`,
```
```suggestion
If you don't check |checkbox| :guilabel:`Enable Service capabilities`,
```

DelazJ commented on docs/server_manual/getting_started.rst:
```diff
@@ -810,6 +810,9 @@ Configure your project
 To provide a new QGIS Server WMS, WFS or WCS, you have to create a QGIS project
```
```suggestion
To provide a new QGIS Server WMS, WFS, OAPIF or WCS, you have to create a QGIS project
```

DelazJ reviewed (commented):
> which part of the text do you mean where WFS tab needs remaining. 

OAPIF is now part of the tab name
